python/cudf/cudf/core/multiindex.py

```diff
@@ -20,22 +20,75 @@ class MultiIndex(Index):
 
     Provides N-Dimensional indexing into Series and DataFrame objects.
 
-    Properties
-    ---
-    levels: Labels for each category in the index hierarchy.
-    codes: Assignment of individual items into the categories of the hierarchy.
-    names: Name for each level
+    Parameters
+    ----------
+    levels : sequence of arrays
+        The unique labels for each level.
+    labels : sequence of arrays
+        labels is depreciated, please use levels
+    codes: sequence of arrays
+        Integers for each level designating which label at each location.
+    sortorder : optional int
+        Not yet supported
+    names: optional sequence of objects
+        Names for each of the index levels.
+    copy : bool, default False
+        Copy the meta-data. Not yet supported
```

Comment:
We are checking the copy parameter below but this docstring says it's not yet supported

Reply:
missed it, will update.